### PR TITLE
refactor: Generate jwk qr code

### PR DESCRIPTION
**Pull request by pawanpaudel93**
## Summary

This PR updates the UI to generate the wallet JWK animated QR Code instead of the address QR Code to be scannable from the mobile app.

This PR must be tested with ArConnect Mobile PR: https://github.com/arconnectio/ArConnect-Mobile/pull/12

### CODE REVIEW

**Review by nicholaswma — APPROVED** [review]
tACK! 🚀

File: src/routes/popup/settings/wallets/[address]/qr.tsx [hunk 1]
```diff
@@ -1,26 +1,226 @@
...
+    } catch {
+    } finally {
+      setLoading(false);
+    }
+  }
+
+  useEffect(() => {
+    if ((wallet as any)?.keyfile) {
+      setFrames(dataToFrames(JSON.stringify((wallet as any)?.keyfile)));
+      freeDecryptedWallet((wallet as any).keyfile);
```
Comment (7i7o) [inline src/routes/popup/settings/wallets/[address]/qr.tsx:89]: The same way we use `freeDecryptedWallet` after using it, do we have a way to remove the frames from memory after it's been used?

**Review by 7i7o — APPROVED** [review]
tACK 👍

File: src/routes/popup/settings/wallets/[address]/qr.tsx (hunk 1 above)
Comment (pawanpaudel93) [reply src/routes/popup/settings/wallets/[address]/qr.tsx:89]: @7i7o Frames are cleared from memory on unmount: https://github.com/arconnectio/ArConnect/pull/451/commits/932fa08a5ed01e7d6c434e0f210c8361342208c0

**Review by 7i7o — APPROVED** [review]
tACK 👍

**Pull request merged**